data/Store.js

```diff
@@ -406,7 +392,7 @@ export class Store {
     }
 
     getRootSummary(rawData) {
-        return this._loadRootAsSummary && rawData.length === 1 && !isEmpty(rawData[0].children) ?
```

Comment:
I removed this as it would cause issues (duplicate row added to grid) if the app provided update data with root summary data which had no children (so only the summary data changed). Don't think this extra condition is necessary.